src/DotLiquid/Properties/AssemblyInfo.cs

```diff
@@ -1,13 +1,13 @@
-﻿using System.Reflection;
+using System.Reflection;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 
-[assembly: AssemblyTitle("DotLiquid")]
-[assembly: AssemblyDescription("DotLiquid is a templating system ported to the .NET framework from Ruby’s Liquid Markup.")]
+[assembly: AssemblyTitle("MIISDotLiquid")]
+[assembly: AssemblyDescription("Fixed version for MIIS - DotLiquid is a templating system ported to the .NET framework from Ruby’s Liquid Markup.")]

```

Comment:
Single customer fix with change name for everyone can't be merged back to master

Reply:
Yes, you're right. Sorry for that. I wouldn't have changed it if I had expected your response, but after a lot of time without it, I decided to go solo. See previos file comment. Thanks.